sirepo/job_supervisor.py

```diff
@@ -193,15 +193,17 @@ def pkdebug_str(self):
         if not d:
             return '_ComputeJob()'
         return pkdformat(
-            '_ComputeJob({} {} ops={})',
+            '_ComputeJob(jid={} uid={} status={} ops={})',
             d.get('computeJid'),
+            d.get('uid'),
             d.get('status'),
             self.ops,
         )
 
     @classmethod
     async def receive(cls, req):
-        pkdlog('{}', req)
+        if req.content.get('api') != 'api_runStatus':
```

Comment:
Is this only place that pkdlog of runStatus happens? If in error, it's fine to log. Just wondering if it is less noisy now.

Reply:
It also happens in [pkcli/job_supervisor.py](https://github.com/radiasoft/sirepo/blob/4e5514742b936dc9fb54b43cafecb3cd14771565/sirepo/pkcli/job_supervisor.py#L169). Eliminating the both of them makes less noise.